UnityProject/Assets/Scripts/Actions/V2/ActionButtonData.cs

```diff
@@ -34,7 +33,7 @@ public class ActionButtonData
 		public bool HasCustomCursor;
 		public CursorOffsetType OffsetType;
 		public Vector2 CursorOffset;
-		public Texture2D CursorTexture;
+		public SpriteDataSO CursorTexture;
```

Comment:
You need to update all the SOs that use a custom texture, I think stuff like Fire built all that other stuff does, If you reverted back to the texture rider will tell you what overrides it, So you can make a note what to update

Reply:
this isn't an SO